install.sh

```diff
@@ -1037,7 +1037,26 @@ download_latest_bundle_and_extract_if_needed() {
     rm -rf $BUILD_DIR
     local URL="https://dl.sandstorm.io/sandstorm-$BUILD.tar.xz"
     echo "Downloading: $URL"
-    curl -A "$CURL_USER_AGENT" -f "$URL" | tar Jxo
+    curl -A "$CURL_USER_AGENT" -f "$URL" > /var/tmp/sandstorm-$BUILD.tar.xz
```

Comment:
+1 to the "Don't use a predictable filename" suggestion, mostly because that means the logic on retry gets harder.

I'm still hopeful we can gracefully handle re-downloading the bundle -- I've seen people's Internet connection flake out mid-download, and using a non-predictable filename would help with that.